Modules/Core/src/Services/SessionsService.php

```
@@ -51,7 +51,7 @@ public function auth($email, $password)
             }
             if ($this->crypt->check_password($user->user_password, $password)) {
                 $session_data = ['user_type' => $user->user_type, 'user_id' => $user->user_id, 'user_name' => $user->user_name, 'user_email' => $user->user_email, 'user_company' => $user->user_company, 'user_language' => $user->user_language ?? 'system'];
-                $this->session->set_userdata($session_data);
+                session([$session_data);
```

PHPStan (2.1.31)

Syntax error, unexpected ')', expecting ']' on line 54


(phpstan.parse)